frontend/src/metabase/lib/query/filter.js

```diff
@@ -66,10 +74,35 @@ export function canAddFilter(filter: ?FilterClause): boolean {
 // FILTER TYPES
 
 export function isStandard(filter: FilterClause): boolean {
+  if (!Array.isArray(filter)) {
+    return false;
+  }
+
+  // undefined args represents an incomplete filter (still standard, but not valid)
+  const isLiteralOrUndefined = arg => (arg ? isLiteral(arg) : true);
+
+  const op = filter[0];
+  const field = filter[1];
+  const args = filter.slice(2);
```

Comment:
Question, very nit-picky but I'm curious…

What would be your view on writing these as 

``` js
const [op, field, …args] = filter
```

Reply:
Sure thing, that'll work, too!